frontend/src/lib/components/report/Report.svelte

```diff
@@ -43,7 +43,7 @@
 	on:focus={() => (hovering = true)}
 	on:mouseleave={() => (hovering = false)}
 	on:blur={() => (hovering = false)}
-	class="border-solid mr-2 mb-2 rounded-lg border-grey-light border shadow-sm flex flex-col p-3 px-5 hover:shadow-md w-full sm:w-80 h-60"
+	class="border-solid rounded-lg border-grey-light border shadow-sm flex flex-col p-3 px-5 hover:shadow-md h-full"
```

Comment:
Moving forward, maybe we could make this card a component so that we don't duplicate code between reports and projects.